src/server/db_slice.h

```diff
@@ -544,8 +546,9 @@ class DbSlice {
   // Used in temporary computations in Acquire/Release.
   mutable absl::flat_hash_set<uint64_t> uniq_fps_;
 
+  mutable util::fb2::SharedMutex cb_mu_;  // to prevent removing callback during call
```

Comment:
can you explain the comment?
during call of what?
currently we do not allow preemption in change_cb_, if they preempt we have a bug probably

Reply:
you are right we have FiberAtomicGuard there

Reply:
You are right. We don't have a bug